internal/metrics/metrics.go

```diff
@@ -73,13 +76,139 @@ var (
 	UpstreamRPCDuration = promauto.NewHistogramVec(
 		prometheus.HistogramOpts{
 			Namespace: metricsNamespace,
-			Subsystem: metricsSubsystem,
+			Subsystem: "router",
 			Name:      "upstream_rpc_duration_seconds",
 			Help:      "Latency of RPC requests forwarded to upstreams.",
-			Buckets:   []float64{0.1, .5, 1, 5, 10, 30, 60},
+			Buckets:   []float64{.005, .01, .025, .05, .1, .25, .5, 1, 2.5, 5, 10, 20, 40},
 		},
 		[]string{"client", "endpoint_id", "url", "jsonrpc_method", "response_code", "jsonrpc_error_code"},
 	)
+
+	// Health check metrics
+
+	BlockHeight = promauto.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: metricsNamespace,
+			Subsystem: "healthcheck",
+			Name:      "block_height",
+			Help:      "Block height of upstream.",
+		},
+		[]string{"endpoint_id", "url"},
+	)
+
+	BlockHeightTotalRequests = promauto.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: metricsNamespace,
+			Subsystem: "healthcheck",
+			Name:      "block_height_total",
```

Comment:
should we include `requests`, like `block_height_requests` to be consistent with the metrics in the subgraph gateway? I also removed `total` from the end. Same w/ the other metrics.